backend/de.metas.acct.base/src/main/java/de/metas/acct/process/ExportAccountInfos.java

```diff
@@ -0,0 +1,159 @@
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+/**
+ * @author metas-dev <dev@metasfresh.com>
+ */
+public class ExportAccountInfos extends JavaProcess
+{
+	final SpreadsheetExporterService spreadsheetExporterService = SpringContextHolder.instance.getBean(SpreadsheetExporterService.class);
+	final IAccountDAO accountDAO = Services.get(IAccountDAO.class);
+
+	@Param(parameterName = "C_AcctSchema_ID", mandatory = true)
+	private int p_C_AcctSchema_ID;
+
+	@Param(parameterName = "DateAcctFrom", mandatory = true)
+	private Timestamp p_DateAcctFrom;
+
+	@Param(parameterName = "DateAcctTo", mandatory = true)
+	private Timestamp p_DateAcctTo;
+
+	@Param(parameterName = "Function", mandatory = true)
+	private String p_Function;
+
+	@Override
+	protected String doIt()
+	{
+		final Evaluatee evalCtx = getEvalContext();
+		final String fileNameSuffix = TimeUtil.asLocalDate(p_DateAcctFrom).toString() + "_" + TimeUtil.asLocalDate(p_DateAcctTo).toString();
+
+		final List<AccountId> accountIds = accountDAO.retrieveAccountsForTimeFrame(AcctSchemaId.ofRepoId(p_C_AcctSchema_ID), p_DateAcctFrom, p_DateAcctTo);
+		final List<File> files = new ArrayList<>();
+
+		for (final AccountId accountId : accountIds)
+		{
+			final String sql = getSql(accountId);
+			final MAccount account = accountDAO.getById(accountId);
+			final String fileName = account.getAlias() + "_" + fileNameSuffix;
+
+			final JdbcExcelExporter jdbcExcelExporter = JdbcExcelExporter.builder()
+					.ctx(getCtx())
+					.translateHeaders(true)
+					.applyFormatting(true)
+					.fileName(fileName)
+					.build();
+
+			spreadsheetExporterService.processDataFromSQL(sql, evalCtx, jdbcExcelExporter);
+
+			final File resultFile = jdbcExcelExporter.getResultFile();
+
+			files.add(resultFile);
+		}
+
+		try
+		{
+			final String zipFileName = TimeUtil.asLocalDate(p_DateAcctFrom).toString() + "_" + TimeUtil.asLocalDate(p_DateAcctTo).toString() + "_";
+			final File zipFile = FileUtil.zip(files, zipFileName);
+			getResult().setReportData(zipFile);
+		}
+		catch (IOException e)
+		{
+			throw new RuntimeException(e);
+		}
+
+		return MSG_OK;
+	}
+
+	private String getSql(final AccountId accountId)
+	{
+
+		final StringBuffer sb = new StringBuffer();
+		sb.append("SELECT * FROM ")
+				.append(p_Function)
+				.append("(")
+				.append("p_Account_ID := ")
+				.append(accountId.getRepoId())
+				.append(", ")
+				.append("p_C_AcctSchema_ID := @C_AcctSchema_ID@, ")
+				.append("p_DateAcctFrom := '@DateAcctFrom@', ")
+				.append("p_DateAcctTo := '@DateAcctTo@')");
```

Comment:
why using context vars like `@C_AcctSchema_ID@`, for params you already have (see p_C_AcctSchema_ID)?